### PR TITLE
Remove token on sign out, reset form

### PR DESCRIPTION
**Pull request by jdungan**
Signing out of codesy now stores "" as the token, which triggers the form to refresh and retrieve the 401 response.

This also includes several refactors to try to better explain what's happening.

This fixes codesy/codesy#130 which should have been an extension issue

### CODE REVIEW

File: src/codesy.coffee [hunk 1]
```diff
@@ -1,22 +1,23 @@
 console.time 'codesy load'
 
 codesy =
-  url : ""
```
Comment (jdungan) [inline src/codesy.coffee]: Saving domain in a variable called url was confusing

File: src/codesy.coffee [hunk 2]
```diff
@@ -40,48 +41,47 @@ codesy.events.submit = (e)->
...
-            console.log "codesy: form request success"
-            codesy.appendForm data
-          .fail (err) ->
-            console.timeEnd "codesy: request form"
-            console.log "codesy: form request failed"
-            if err.status = 401
-              codesy.appendForm err.responseText
-            else
-              console.log err
+    codesy.bid.get {url:window.location.href}
```
Comment (jdungan) [inline src/codesy.coffee]: removed the local storage check;  it now runs separately and then fires new page.

File: src/codesy_home.coffee [hunk 3]
```diff
@@ -1,31 +1,26 @@
...
-      'domain': window.location.href
-      'token': token_found
+codesy={}
+codesy.find = (domains,domain) ->
+  domains.map((item) -> item.domain).indexOf(domain)
+
+
+console.time("codesy: token set");
+
+new_token =
```
Comment (jdungan) [inline src/codesy_home.coffee]: A new domain is now added to local storage on every new page.  This allows the token to be cleared on sign out and eliminates all the if/then checks

**Comment by groovecoder** [comment]
Hmm ... not sure how/why I missed this. Should we keep or close it in light of the pending Firefox PR?

**Comment by groovecoder** [comment]
Superseded by #68

**Pull request closed without merging**